package.json

```diff
@@ -10,7 +10,7 @@
     "test": "test"
   },
   "scripts": {
-    "test": "./test/test.sh $(find test -name '*_test.js') --reporter spec"
+    "test": "./test/test.sh"
```

Comment:
Really?

Jeg run it with commandline instead of modifying the `package.json` unless this make sense in general...